test/blackbox/helpers_delete_images.bash

```diff
@@ -0,0 +1,84 @@
+ROOT_DIR=$(git rev-parse --show-toplevel)
```

Comment:
can't we use an existing helper script?

Reply:
Yes, I think we can

Reply:
I tried to make it similar to the other tests

Reply:
I tried to use other helper scrips and none work for this bats tests, i'm not sure why

Reply:
ok